tests/run.sh

```diff
@@ -64,8 +64,9 @@ PREFIX_TEST_ENVS="SK"
 PREFIX_TEST_TAGS="TAG_"
 REGISTRY_URL=${1:-"spinkubeperf.azurecr.io"}
 TEST=${TEST:-"hello-world"}
-export TAG_TEST_NAME=$TEST
 OUTPUT=${OUTPUT:-"datadog"}
+export TAG_SPIN_VERSION="${SK_OCI_TAG}"
```

Comment:
I don't think we assume these are always the same. For example, for density tests we use `latest`. Maybe we support passing `SPIN_VERSION` to the script and default to `SK_OCI_TAG`?

Reply:
Ah right, thank you.  Updated.